### ISSUE TITLE
Upgrade to MSYS2?

### ISSUE THREAD

**Issue by hakonhagland** [PerlAlien/Alien-MSYS#25]
In view of the issue with the path length limitation of 260 characters for `install.exe` in MSYS, would it be an idea to upgrade this module (or create a new module Alien::MSYS2) to use MSYS2? See https://github.com/PerlAlien/Alien-Build/pull/415#issuecomment-2237108656 for more information

**Comment by plicease** [comment]
Yeah, Alien::MSYS2 would be ideal.  I resisted this before because it was much larger even for a minimal install than MSYS.  But MSYS hasn't been actively developed for years at this point.

**Comment by plicease** [comment]
I started working on this at some point, but maybe it is time to resurrect it:
https://github.com/plerk/Alien-MSYS2

**Comment by shawnlaffan** [comment]
I too would be keen to see and help this happen.

**Comment by plicease** [comment]
Okay I have a Windows 10 machine I can use to test this.  The current version of `Alien::MSYS2` is very out of date with my DZil bundle and as a result not fit to open PRs against, but I will see what I can do to remedy that.
